app/lib/features/chat/widgets/convo_card.dart

```diff
@@ -405,45 +408,108 @@ class _SubtitleWidget extends ConsumerWidget {
             ),
           ],
         );
-      case 'm.room.member':
-        MsgContent? msgContent = eventItem.msgContent();
-        if (msgContent == null) {
-          return const SizedBox();
-        }
-        String body = msgContent.body();
-        String? formattedBody = msgContent.formattedBody();
-        if (formattedBody != null) {
-          body = simplifyBody(formattedBody);
-        }
+      case 'MembershipChange':
```

Comment:
considering that we are doing the changes for chat-ng, I wonder if we have to do these changes here. With the new format, it will just go to custom events for chat then? That's fine, let's just ignore that then. I don't care that these new objects are not supported in legacy chat.

Reply:
When you PR is ready, please let me know.
I will review and fix the membership change & profile change.